pkg/services/correlations/models.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 var (
-	ErrSourceDataSourceReadOnly           = errors.New("source data source is read only")
+	ErrCorrelationsReadOnly               = errors.New("correlation is read only")
```

Comment:
maybe we should use the singular form?
```suggestion
	ErrCorrelationReadOnly               = errors.New("correlation is read only")
```

Reply:
Good call! Updated in 233e13fb10e5c76d1234a2430377e2bddff11095